bigtable/client/filters.h

```diff
@@ -200,10 +200,13 @@ class Filter {
    * of multiple filters via the Union() function (aka Interleaved in the
    * proto).  Furthermore, notice that this is the cells within a row, if there
    * are multiple column families and columns, the cells are returned ordered
-   * by first column family, and then by column qualifier, and then by
-   * timestamp.
+   * by:
+   * - The column family internal ID, which is not necessarily the
+   *   lexicographical order of the column family names.  Also, the ID of each
+   *   column family may change on each row.
```

Comment:
I'm not quite sure how to parse this. Is there a way to rephrase? Overall the description seems accurate, though.